### PR TITLE
Redux State Management

### PR DESCRIPTION
**Pull request by Seun-A**

### CODE REVIEW

File: src/components/box/box.component.js [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
-        day === 'Sun' && index === 0 ? 'rounded-bl-3xl':
-        day === 'Sun' && index === 2 ? 'rounded-br-3xl': null
-      }
+      
+      border-b-2
+      group-[&:nth-child(8)]:border-b-0
+      border-e-2
+      [&:nth-child(4)]:border-e-0
+
+      group-[&:nth-child(2)]:[&:nth-child(2)]:rounded-tl-3xl
```
Comment (mofopeojosh) [inline src/components/box/box.component.js:36]: nice!

File: src/components/form-create/form-create.component.js [hunk 2]
```diff
@@ -1,51 +1,75 @@
...
-      setAlert(false)
+      resetCreateForm()
     } else {
       setAlert(true)
     }
   }
 
   const handleChange = event => {
     const { value, name } = event.target
+    console.log(value)
```
Comment (mofopeojosh) [inline src/components/form-create/form-create.component.js]: remove console.log

File: src/components/sidebar/sidebar.component.js [hunk 3]
```diff
@@ -10,10 +11,10 @@ const Sidebar = ({ editFormVisible, ...props }) => (
     `}
   >
     {
-      editFormVisible ? 
-        <EditForm {...props} />
+      useSelector(state => state.edit).visible ?
```
Comment (mofopeojosh) [inline src/components/sidebar/sidebar.component.js]: better to assign this to a variable to keep your template clean
```suggestion
      isVisible ? 
```

File: src/components/text-area/text-area.component.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
 const TextArea = ({ ...props }) => (
   <textarea
     {...props}
-    name="boxContent"
+    name="content"
+    id='text-area'
```
Comment (mofopeojosh) [inline src/components/text-area/text-area.component.js]: You can leave this for now but ideally it's not a good practice to add `id`s to reusable components.
If you had two textareas on the same page, you'd have two elements with the same id but that shouldn't be so https://www.w3schools.com/html/html_id.asp

**Review by mofopeojosh — APPROVED** [review]
You did a great job here! 🚀

File: src/components/form-create/form-create.component.js (hunk 2 above)
Comment (Seun-A) [reply src/components/form-create/form-create.component.js]: Thank you!
Done

File: src/components/text-area/text-area.component.js (hunk 4 above)
Comment (Seun-A) [reply src/components/text-area/text-area.component.js]: I found a way to successfully trigger the focus in the box component.
The id **'text-area'** was changed under this effect to **'edit-text-area'**

> in [src/components/box/box.component.js](https://github.com/Seun-A/weekly-meal-planner/pull/12/files/39556b29107681b509f19f2f554f78eb20732038#diff-9e48d92febadff0dcaa42367db7a519b28067a23f0399784535eaa6535fbc2b4)

`14  setTimeout(() => { document.getElementById('edit-text-area').focus() }, 170); `

I plan to modify this later

File: src/components/sidebar/sidebar.component.js (hunk 3 above)
Comment (Seun-A) [reply src/components/sidebar/sidebar.component.js]: Done

**Pull request merged**